e2e/_suites/fleet/ingest-manager_test.go

```diff
@@ -43,6 +43,10 @@ var agentVersionBase = "8.0.0-SNAPSHOT"
 // It can be overriden by ELASTIC_AGENT_VERSION env var
 var agentVersion = agentVersionBase
 
+// agentStaleVersion is the version of the agent to use as a base during upgrade
+// It can be overriden by ELASTIC_AGENT_STALE_VERSION env var. Using latest GA as a default.
+var agentStaleVersion = "7.10.0"
```

Comment:
Could you add it to the Jenkinsfile as an input parameter? This way we will allow manual tests on CI simply setting that input parameter in Jenkins UI

Reply:
Adding it to Jenkins as an optional param is even better... where it uses the value if passed... but if not, we can do some parse and math logic to get the current-minus-1 version (from the stack deployed or other e2e-testing variable) and use that instead.  It is more future proof and will require less maintenance, right?  

It is worth some discussion, open to what ever you want to merge, and if not tackled now, we can just drop a ticket in for the reminder of the work.